chapter_convolutional-modern/alexnet_vn.md

```diff
@@ -103,7 +118,12 @@ In the case of an image, the lowest layers might come to detect edges, colors, a
 Indeed, :cite:`Krizhevsky.Sutskever.Hinton.2012` proposed a new variant of a convolutional neural network which achieved excellent performance in the ImageNet challenge.
 -->
 
-*dịch đoạn phía trên*
+Một nhóm nhà nghiên cứu khác, bao gồm Yann LeCun, Geoff Hinton, Yoshua Bengio, Andrew Ng, Shun-ichi Amari, và Juergen Schmidhuber, đã có những kế hoạch khác. 
+Họ tin rằng các đặc trưng tự chúng có thể được học. 
```

Comment:
```suggestion
Họ tin rằng các đặc trưng  nên tự được học lấy. 
```

Reply:
Cái câu này như mình hiểu ý tác giả là các đặc trưng tự nó đã đủ để nói về nó mà không cần xây một model biểu diễn phức tạp nào từ cách nhìn của con người. Học từ chính data thay vì biểu diễn dưới dạng mô hình toán nào đó như trước đây

Reply:
E cũng hiểu giống a mà, em chỉ góp ý nghe cho hợp hơn, tại vì chữ `ought to` anh ko dịch. Chỗ này ko nói về việc có thể học được hay ko mà là có nên học hay ko hay tính thủ công tiếp.

Reply:
Ở đây mình dùng từ "có thể" cho từ "ought to" để diễn tả một dự đoán. Có lẽ để rõ hơn thì nên dịch là "Họ tin rằng các đặc trưng tự chúng có thể được hệ thống học máy học được".